internal/performance/performance.go

```diff
@@ -125,60 +125,101 @@ func (c *PerfCollector) Collect(mos []types.ManagedObjectReference, metrics []ty
 	return perfMetricsByRef
 }
 
-type Accumulator struct {
+// The metric returned have a field indicating the 'instance' it refers to. However, the vCenter could mix standard values and aggregated values. We give priority
+// to the raw values and fall back to 'instanceless' values in case no raw data has been received
```

Comment:
this sentence is a bit hard to understand to me. would make sense to simplify it with something la `instance metrics take precedence over average calculated ones` ?